### ISSUE TITLE
Links to Firmware on Getting Started page lead to 404 error

### ISSUE THREAD

**Issue by kampfhamster309** [emsec/ChameleonMini#307]
I just got my chameleon and tried to follow the getting started page. Unfortunately, the links on it seem to be broken/old. If i try to access 

https://cdn.statically.io/gh/emsec/ChameleonMini/master/Firmware/Chameleon-Mini/Latest/Chameleon-Mini.hex

or

https://cdn.statically.io/gh/emsec/ChameleonMini/master/Firmware/Chameleon-Mini/Latest/Chameleon-Mini.eep

to grab the new Firmware -> all I get is a 404 Error. It would be great if you could update those links or at least remove them.

**Comment by fptrs** [comment]
Hi @kampfhamster309,
I checked the getting [started page](https://cdn.statically.io/gh/emsec/ChameleonMini/master/Doc/Doxygen/html/Page_GettingStarted.html) and it correctly refers to the [release section](https://github.com/emsec/ChameleonMini/releases). Can you tell me where you found the links?

**Comment by kampfhamster309** [comment]
It looks to me like there is an older version of the getting started page hanging around... 

If I look at the README.md in this very repo

<img width="873" alt="Bildschirmfoto 2021-11-23 um 13 18 38" src="https://user-images.githubusercontent.com/17223013/143023178-7117c7e9-9432-45e9-9c19-0f8668d0082a.png">

and click on the "getting started" link, I land on another version than you linked above.

There, the step 2 part looks like this:

<img width="1334" alt="Bildschirmfoto 2021-11-23 um 13 19 05" src="https://user-images.githubusercontent.com/17223013/143023404-a088ef2b-d050-4c69-8384-278b20183789.png">

The release section is not mentioned there, and the two links to the *.hex and *.eep files are the ones I posted above, which lead to the 404 error. 

I ended up just building the files myself from source, but maybe some more inexperienced people would benefit if you update the README.md to the correct getting started section.
